src/ONE.Models.CSharp/ONE.Models.CSharp.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
-    <VersionPrefix>4.14.0</VersionPrefix>
+    <VersionPrefix>4.15</VersionPrefix>
     <!--For Feature packages update the VersionSuffix with alpha and an incremental number. E.g. alpha1, alpha2...-->
     <VersionSuffix></VersionSuffix>
```

Comment:
Should we be doing a pre-release for this package?

Reply:
We won't directly make functional change, which is a separate story, therefore, no meaningful to do a pre-release.